qt_binder/tests/test_binder.py

```diff
@@ -59,7 +59,11 @@ def test_object_traits(self):
                   for k, v in self.Object.class_traits().items()}
         self.assertIsInstance(traits['deleteLater'], QtSlot)
         self.assertIsInstance(traits['destroyed'], QtSignal)
-        self.assertIsInstance(traits['destroyed_QObject'], QtSignal)
+        try:
+            self.assertIsInstance(traits['destroyed_QObject'], QtSignal)
+        except KeyError:
+            # on Python 3, key name is different
+            self.assertIsInstance(traits["destroyed_b'QObject*'"], QtSignal)
```

Comment:
I am using the following PyQt4/Qt4 versions (installed via anaconda)
pyqt 4.11.4
qt 4.8.7